Symfony/CS/Fixer/PSR2/FunctionCallSpaceFixer.php

```diff
@@ -88,13 +88,13 @@ private function fixFunctionCall(Tokens $tokens, $index)
     }
 
     /**
-     * Gets the name of tokens which can work as function calls.
+     * Gets the token kinds which can work as function calls.
      *
-     * @staticvar string[] $tokens Token names.
+     * @var int[] $tokens Token kinds.
```

Comment:
The idea of `@var` is only to indicate the type. That's why they originally went for `@type`, then had to go back again. The staticness is meant to be known by the actual php code. No need to define something twice basically.